client/packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -161,6 +161,8 @@ const OutboundShipmentListViewComponent: FC = () => {
           header: '',
           enableColumnActions: false,
           enableSorting: false,
+          enableResizing: false,
```

Comment:
This column leaves a weird floating handle (since the header string is empty), so easiest to just turn off this property.